fundamentals/middleware/problem-details-service/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ProblemDetailsWebApi.Controllers;
+
+[Route("api/[controller]/[action]")]
+[ApiController]
+public class ValuesController : ControllerBase
```

Comment:
@sammychinedu2ky  PLEASE review controller. You can probably help clean it up.

Reply:
Ok sure let me look into it